src/config/lang/english.go

```diff
@@ -241,7 +241,7 @@ const (
 	CmdPackageDeployFlagSet                    = "Specify deployment variables to set on the command line (KEY=value)"
 	CmdPackageDeployFlagComponents             = "Comma-separated list of components to install.  Adding this flag will skip the init prompts for which components to install"
 	CmdPackageDeployFlagShasum                 = "Shasum of the package to deploy. Required if deploying a remote package and \"--insecure\" is not provided"
-	CmdPackageDeployFlagSget                   = "Path to public sget key file for remote packages signed via cosign"
+	CmdPackageDeployFlagSget                   = "(DEPRECATED) Path to public sget key file for remote packages signed via cosign. This flag will be removed in v0.29.0, please use the --key flag instead."
```

Comment:
```suggestion
	CmdPackageDeployFlagSget                   = "(DEPRECATED) Path to public sget key file for remote packages signed via cosign. This flag will be removed in v0.29.0, please use the --key flag instead."
```
We should add `[Deprecated]` to the front of the description for `CosignKeyPath` as well in `src/types/Component.go` (it must be `[Deprecated]` there for the schema generation to pickup the right thing)